mixins/global.js

```diff
@@ -1,6 +1,6 @@
 import swal from 'sweetalert';
 
-import { SUCCESS, ERROR, SWALCOLORS } from '@/constants.js';
+import { SUCCESS, SWALCOLORS } from '../constants.js';
```

Comment:
Why are we moving back to static path system?

Reply:
Adding relative path throughs error on test.

Reply:
resolved

Reply:
resolved